### PR TITLE
add support for lidar and lidar_seq

### PR DESCRIPTION
**Pull request by fredrikjacobson**
Adds support for:
- `lidars`
- `lidars_sequence`

### CODE REVIEW

**Comment by cotramarko** [comment]
Great! 🚀  Let's make sure to update the gh-pages docs as well before we release this :)

**Review by cotramarko — APPROVED** [review]

File: annotell-input-api/annotell/input_api/resources/input/lidars.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+               dryrun: bool = False) -> Optional[CreateInputResponse]:
+        """
+        Upload files and create an input of type ``Lidars``.
+
+        :param lidars: class containing 3D resources that constitute the input
+        :param project: project to add input to
+        :param batch: batch, defaults to latest open batch
+        :param annotation_types: annotation types for which to produce annotations for. Defaults to `None` (corresponds to all available annotation types). Passing an empty list will result in the same behaviour as passing `None`.
+        :param dryrun: If True the files/metadata will be validated but no input job will be created.
+        :returns InputJobCreated: Class containing id of the created input job, or `None` if dryrun.
```
Comment (Ragge-dev) [inline annotell-input-api/annotell/input_api/resources/input/lidars.py:28]: InputJobCreated -> CreateInputResponse

File: annotell-input-api/annotell/input_api/resources/input/lidars_sequence.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+               dryrun: bool = False) -> Optional[input_model.CreateInputResponse]:
+        """
+        Upload files and create an input of type ``LidarsSequence``.
+
+        :param lidars_sequence: class containing 3D resources that constitute the input
+        :param project: project to add input to
+        :param batch: batch, defaults to latest open batch
+        :param annotation_types: annotation types for which to produce annotations for. Defaults to `None` (corresponds to all available annotation types). Passing an empty list will result in the same behaviour as passing `None`.
+        :param dryrun: If True the files/metadata will be validated but no input job will be created.
+        :returns InputJobCreated: Class containing id of the created input job, or `None` if dryrun.
```
Comment (Ragge-dev) [inline annotell-input-api/annotell/input_api/resources/input/lidars_sequence.py:28]: InputJobCreated -> CreateInputResponse

**Review by Ragge-dev — APPROVED** [review]
Great work! I found some very small errors in the doc-string, fix that and I think you're good to go

**Pull request merged**